lib/Braintree/Logging/CurlAuditLoggerInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace Braintree\Logging;
+
+/**
+ * Raised when authentication fails.
+ * This may be caused by an incorrect Configuration
+ *
+ * @package    Braintree
+ * @subpackage Logging
+ */
+interface CurlAuditLoggerInterface
+{
+
+
+    public function log();
+
+    /**
+     * @param $responseTime
```

Comment:
For all the methods defined in this file, could you document what they are expected to do, and add some additional documentation for each variable?